public/js/report.js

```diff
@@ -742,6 +745,24 @@ var Hit = React.createClass({
 });
 
 /**
+ * Renders Length Distribution of all hits per query
+ */
+var LengthVis = React.createClass({
+  plot_svg: function () {
+    return $(React.findDOMNode(this.refs.plot_test));
+  },
+  componentDidMount: function () {
+    this.plot_test = new LengthVisualisation(this.props.query, this.plot_svg(), this.props.algorithm);
+  },
+  render: function () {
+    return(
+      <div ref="plot_test">
```

Comment:
I would prefer the name "svgContainer". Also, give it a class: length-distribution.